docs/explain-mmlu.md

```diff
@@ -0,0 +1,17 @@
+1) **Main point**
+
+Obtain a synthetic dataset (answers + brief explanations + analysis of erroneous answers + CoT tokens) for training subsequent models.
```

Comment:
We actually want to elicit the full reasoning chain, don't we?

Could you also add why we want to do it? AFAIU, we want to fine-tune small models on different versions of the distilled CoT and compare the performance. Right?